sdk/messaging/azservicebus/receiver.go

```diff
@@ -516,3 +527,11 @@ func createLinkOptions(mode ReceiveMode, entityPath string) []amqp.LinkOption {
 
 	return opts
 }
+
+func checkReceiverMode(receiveMode ReceiveMode) error {
+	if receiveMode == ReceiveModePeekLock || receiveMode == ReceiveModeReceiveAndDelete {
+		return nil
+	} else {
```

Comment:
Nit, unnecessary `else` (the linter should be catching these, need to check).

Reply:
Fixed.